src/ParamConverter/ShowCardParamConverter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ParamConverter;
+
+use App\Repository\CardRepository;
+use App\Request\ShowCard;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class ShowCardParamConverter implements ParamConverterInterface
+{
+    private CardRepository $cardRepository;
+
+    public function __construct(CardRepository $cardRepository)
+    {
+        $this->cardRepository = $cardRepository;
+    }
+
+    public function apply(Request $request, ParamConverter $configuration)
+    {
+        $card = $this->cardRepository->findOneById((int)$request->get('id'));
```

Comment:
Hmmm this is unnecessary here now. 🤔  Here we should make raw validation on request parameters in current case. So make here basic validation, eg check if id is correct value. Eg: "Id must be grater or equal 1." etc